Sources/SmileID/Classes/Networking/ServiceRunnable.swift

```diff
@@ -202,11 +232,26 @@ extension ServiceRunnable {
             throw URLError(.badURL)
         }
 
+        let encoder = JSONEncoder()
+        encoder.outputFormatting = [.sortedKeys, .withoutEscapingSlashes]
+        var signedHeaders = headers
+        do {
+            let payload = try encoder.encode(body)
+            if let header = headers.first(where: { $0.name == "SmileID-Request-Timestamp" }) {
+                let requestMac = try SmileIDCryptoManager.shared.sign(
+                    timestamp: header.value,
+                    headers: headers.toDictionary(),
+                    payload: payload
+                )
+                signedHeaders += [.requestMac(value: requestMac)]
+            }
+        } catch { /* in case we can't add the security info the backend will deal with the enrollment */ }
```

Comment:
Same here. do-catch block seems to be redundant.

Reply:
Resolved in [f1c0a73](https://github.com/smileidentity/ios/pull/286/commits/f1c0a7342480f07515b317fc20349c7a7ddf0bd5)